envoy-control-services/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/services/ClusterState.kt

```diff
@@ -7,7 +7,8 @@ enum class Locality {
 data class ClusterState(
     val servicesState: ServicesState,
     val locality: Locality,
-    val zone: String
+    val zone: String // TODO(dj): #110 perhaps just call it cluster:
+    // consul => dc, envoy => zone, envoy-control => cluster ?
```

Comment:
Why would you want to call it cluster? There is already cluster in envoy that would add another conflicting name.

Reply:
@chemicL - please add some clarification here and we should move this PR forward.

Reply:
I'm assuming three domains:

* Consul: datacenter, service, check
* Envoy: zone, cluster, endpoint
* Envoy-Control: zone/cluster/?, service, instance

Let me know if you prefer **zone** or **cluster** or have a better idea. My personal favourite is **cluster** simply because **in the context** of a Control Plane I'm assuming a setup where each data center / cloud region per a specific environment has their own Control Plane.

Therefore, (in e-c's context) a cluster is a group of nodes that have their collection of service instances, but services can span multiple clusters.

The main point I'm making here is that it's not useful to consider a particular term without the proper context as the names already appear in two contexts (service/datacenter both used in Consul and in e-c). So far they've covered the same concept which is fortunate. 

But, due to the fact that in one data center we can have many clusters, instead of **datacenter** I'd prefer to use the term **cluster** from now on. If there's a nomenclature we can use that is distinct from Consul's and from Envoy's that would be great, but I can't think of anything familiar that would avoid confusion. These names are just natural to me and probably to the rest of us.

Reply:
I would be much happier with `zone`.